.changeset/itchy-feet-poke.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+`Tabs` - Added `@size` argument with new "large" size variant
```

Comment:
@didoo Would this still be a minor?

Would I just add a "major" update for just the tokens since I added new ones and changes the names of some existing ones related to the size variants?

Reply:
Added "major" bump for token changes. (In the process of editing to list out changes in detail.)